### ISSUE TITLE
v1.0 demo

### ISSUE THREAD

**Issue by magg** [IBM-Blockchain/car-lease-demo#95]
hello

is this demo compatible with fabric version 1.0

**Comment by mrshah-at-ibm** [comment]
No, this demo is not yet upgraded to fabric version 1.0

**Comment by FollowJack** [comment]
Is there a plan to make it compatible with 1.0?

**Comment by mqmatt** [comment]
Yes there is - we're largely waiting on V1 to be stabilised but it will be updated.

**Comment by AshNanjangud** [comment]
Is the demo compatible with fabric 1.0 now

**Comment by mrshah-at-ibm** [comment]
nope, it is not

**Comment by mastersingh24** [comment]
No and likely this particular demo will be archived.  There's current a sample (it does not have a UI) which is loosely based on the car lease demo that you can read about here:   http://hyperledger-fabric.readthedocs.io/en/latest/write_first_app.html

**Comment by mqmatt** [comment]
I can recommend the newer Vehicle Lifecycle demo, which is based on Fabric V1 and written using Composer: https://github.com/hyperledger/composer-sample-applications/tree/master/packages/vehicle-lifecycle
